target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -21,6 +21,12 @@ case "$board" in
 "glinet,ar150")
 	ucidef_set_led_wlan "wlan" "WLAN" "gl-ar150:orange:wlan" "phy0tpt"
 	;;
+"tplink,tl-wdr3600"|\
+"tplink,tl-wdr4300")
+	ucidef_set_led_usbdev "usb1" "USB1" "tp-link:green:usb1" "1-1.1"
+	ucidef_set_led_usbdev "usb2" "USB2" "tp-link:green:usb2" "1-1.2"
```

Comment:
put these into dts directly please.

Reply:
Did you have a hint for me how to achieve this inside dts for the two ports ?

Reply:
I had an email with this:

I've a request. Please don't use ucidef_set_led_usbdev but put info
about USB ports & LEDs in DT instead. For a reference you may check:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=69d22c70ac9ad66be671ad2517ad5ee41058255f
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=0b1f11002a70abf958aacbf4c18858443f504986

Reply:
Great, does work... Thanks!